com.github.maoschanz.DynamicWallpaperEditor.json

```diff
@@ -0,0 +1,48 @@
+{
+    "app-id" : "com.github.maoschanz.DynamicWallpaperEditor",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "dynamic-wallpaper-editor",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home:rw"
+    ],
+    "build-options" : {
+        "cflags" : "-O2 -g",
+        "cxxflags" : "-O2 -g",
+        "env" : {
+            "V" : "1"
+        }
+    },
+    "cleanup" : [
```

Comment:
Only include what you use instead of just copy pasting a broad list.

Reply:
It's the default manifest provided by Builder, i just changed the uuid and the URI, added home:rw, and removed dconf, i would really enjoy do include "only what i use" but i don't really understand most of other things (and "if it ain't broke...") 🙁

Reply:
> but i don't really understand most of other things (and "if it ain't broke...") slightly_frowning_face

That really isn't an attitude I want from a package maintainer.

The `cleanup` list is files deleted from the package at the end of the build. Since there is only one module and its yours you should understand which ones apply to you.

Reply:
>That really isn't an attitude I want from a package maintainer.

Well it's my very first app, so i'm aware i've things to learn

I saw what it does in the doc, but i'm not sure where these files appears if they do, so i don't know if there is something to clean: i thought it was better to clean potentially inexisting files than not cleaning existing ones.

Reply:
Builder hides some details so I'm not sure the easiest way to look at the build dir. I guess open `~/.cache/gnome-builder`